packages/subgraph/src/util.ts

```diff
@@ -4,15 +4,27 @@ import { Account } from "../generated/schema";
 export const BIGINT_ONE = BigInt.fromI32(1);
 export const BIGINT_ZERO = BigInt.fromI32(0);
 export const SHORT_HELPER_ADDR = getShortHelperAddr(dataSource.network());
+const MAINNET_SHORT_HELPER_ADDR = Address.fromString(
+  "0x3b4095D5ff0e629972CAAa50bd3004B09a1632C5"
+);
+const ROPSTEN_SHORT_HELPER_ADDR = Address.fromString(
+  "0x8903918DFE74476E90B63061E5b9c3E63b65d3F4"
+);
+const GOCHAIN_SHORT_HELPER_ADDR = Address.fromString(
```

Comment:
what is GOCHAIN? 

Also do we want a constants file? Are there lots of constants we use in subgraph? Might be good to start one just to keep us organized

Reply:
Oh sorry I shld put localhost for this 

And ok can have a constant file